src/views/Dashboard/components/DashboardCards/SavedFilters/index.tsx

```diff
@@ -48,35 +89,35 @@ const SavedFilters = ({ id, key, className = '' }: DashboardCardProps) => {
         />
       }
       content={
-        <List<TUserSavedFilter>
-          className={styles.savedFiltersList}
-          key="2"
-          bordered
-          locale={{
-            emptyText: fetchingError ? (
-              <CardErrorPlaceholder
-                title="Failed to Fetch Saved Filters"
-                subTitle={
-                  <Text>
-                    Please refresh and try again or{' '}
-                    <ExternalLink href={`mailto:${SUPPORT_EMAIL}`}>
-                      <Text>contact our support</Text>
-                    </ExternalLink>
-                    .
-                  </Text>
-                }
-              />
-            ) : (
-              <Empty
-                imageType="grid"
-                description={intl.get('screen.dashboard.cards.savedFilters.noSavedFilters')}
-              />
-            ),
-          }}
-          dataSource={fetchingError ? [] : savedFilters}
-          loading={isLoading}
-          renderItem={(item) => <SavedFiltersListItem id={item.id} data={item} />}
-        />
+        <Tabs
+          className={cx(styles.setTabs, 'navNoMarginBtm')}
+          defaultActiveKey={SavedFilterTag.PARTICIPANT}
```

Comment:
not mandatory, but you could get the counts in one pass (to test/modify if you re going to use it)
```
savedFilters.reduce((xs, x) => {
    return {
    [x.tag]: xs[x.tag] + 1,
   ...xs
    }
}, {[SavedFilterTag.PARTICIPANT]: 0, [SavedFilterTag.VARIANT]: 0})
```

Reply:
I prefer a less performant code but more readable. And since the array ain't gonna be that much, I prefer readability in this case if you don't mind